web/package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "homepage": ".",
   "dependencies": {
+    "@openshift-console/dynamic-plugin-sdk": "^0.0.18",
```

Comment:
I don't think you need the "@openshift-console/dynamic-plugin-sdk" dependency.

Reply:
fixed